docs/en/source/quickstart/quickstart_in_one_minute.ipynb

```diff
@@ -11,33 +11,25 @@
     "\n",
     "## Prerequisite\n",
     " - Ubuntu: `18.04`\n",
-    " - Python: `3.7.x`\n",
-    " - Flask: `1.1.2`\n",
-    " - MindSpore: `CPU-1.1.1`\n",
-    " - TinyMS: `0.1.0`\n",
-    " - numpy: `1.17.5`\n",
-    " - Pillow: `8.1.0`\n",
-    " - pip: `21.0.1`\n",
-    " - requests: `2.18.4`\n",
+    " - docker: `v18.06.1-ce`\n",
+    " - Jupyter: Using tinyms 0.3.1-Jupyter,You can refer to [Docker pull tinyms image](https://tinyms.readthedocs.io/en/latest/quickstart/install.html) to deploy the environment.\n",
```

Comment:
Hi, @xinwenh , please synchronize your changes on the Chinese version with the English version